src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -132,6 +132,7 @@ def add_clone_source(self, volname, subvolume, snapname, flush=False):
             self.metadata_mgr.update_section("source", "group", subvolume.group_name)
         self.metadata_mgr.update_section("source", "subvolume", subvolume.subvol_name)
         self.metadata_mgr.update_section("source", "snapshot", snapname)
+
```

Comment:
It's really helpful in reading code, especially when it's first time, to separate codeblocks by a newline. But since you don't want, I've  removed it.